mlflow/models/evaluation/validation.py

```diff
@@ -40,15 +40,16 @@ class MetricThreshold:
                                   is better; metric value <= baseline model metric value - 1e-10 if
                                   lower is better.
 
-    :param higher_is_better: A required boolean representing whether higher value is
-                             better for the metric.
```

Comment:
Instead of removing `higher_is_better`, can we mark this parameter as deprecated and say what's recommended now?

Reply:
:param higher_is_better: Deprecated. Use `greater_is_better` instead.
                             A required boolean representing whether higher value is
                             better for the metric.